test/test.options.defaults.given.js

```diff
@@ -8,15 +8,19 @@ var program = require('../')
 program
   .version('0.0.1')
   .option('-a, --anchovies', 'Add anchovies?')
-  .option('-o, --onions', 'Add onions?', true)
+  .option('-o, --onions', 'Add onions?')
+  .option('-O, --no-onions', 'No onions')
+  .option('-t, --tomatoes', 'Add tomatoes?', false)
+  .option('-T, --no-tomatoes', 'No tomatoes')
   .option('-v, --olives', 'Add olives? Sorry we only have black.', 'black')
   .option('-s, --no-sauce', 'Uh… okay')
   .option('-r, --crust <type>', 'What kind of crust would you like?', 'hand-tossed')
   .option('-c, --cheese [type]', 'optionally specify the type of cheese', 'mozzarella');
 
-program.parse(['node', 'test', '--anchovies', '--onions', '--olives', '--no-sauce', '--crust', 'thin', '--cheese', 'wensleydale']);
+program.parse(['node', 'test', '--anchovies', '--onions', '--tomatoes', '--olives', '--no-sauce', '--crust', 'thin', '--cheese', 'wensleydale']);
```

Comment:
Can you please add a few tests involving negated options?

E.g., what's the expected behavior if someone specifies a default for `--no-sauce`?

Or something arguably invalid/redundant, e.g. for `--no-onions`, but not `--onions`, or for both `--tomatoes` and `--no-tomatoes`? Thanks!

Reply:
Good comment, I only slightly expanded the tests, and do want to improve the coverage. I will look at that.

Honestly, I don't have an expected behaviour for a default value for a negative option! But that also reminds me I want to add to the README now other changes have landed, to cover the intended usage.